ogre2/src/media/materials/programs/depth_camera_fs.glsl

```diff
@@ -139,7 +139,7 @@ void main()
   }
 
   // clamp xyz and set rgb to background color
-  if (point.x > far - tolerance)
```

Comment:
can you make the same change to `depth_camera_final_fs.glsl`?

Reply:
If I change this, the tests start falling again. expecting -inf and getting inf https://github.com/ignitionrobotics/ign-rendering/blob/lobotuerk/depthRangeChange/test/integration/depth_camera.cc#L324-L326

Reply:
Changing both that and https://github.com/ignitionrobotics/ign-rendering/blob/lobotuerk/depthRangeChange/test/integration/depth_camera.cc#L309-L311 to`maxVal` make all tests pass, but I don't think that makes sense

Reply:
I think the problem is that because `point` was previously clamped to `-inf` so `length(point)` becomes `+inf`. So in `depth_camera_final_fs.glsl` we can change to something like:

```
if (!isinf(point.x) && length(point) > far - tolerance)
```

which just skips this additional clamping if it's already been clamped in the previous pass.

Reply:
34267e4 should take care of this